sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -450,7 +450,7 @@ case class IntegralDivide(left: Expression, right: Expression) extends DivModLik
     Examples:
       > SELECT 2 _FUNC_ 1.8;
        0.2
-      > SELECT MOD(2, 1.8);
+      > SELECT _FUNC_(2, 1.8);
```

Comment:
This will be replaced by `%(2, 1.8)`. Probably, need to add to ignore list as well.